includes/compatibility/abstract-wc-data-compatibility.php

```diff
@@ -204,4 +218,35 @@ public static function delete_meta_data($object, $key)
             delete_post_meta($object_id, $key);
         }
     }
+
+    /**
+     * Fix data stored as objects or serialized strings. Converts everything to array and updates it in the meta data.
+     *
+     * @param \WC_Data $object
+     * @param string   $key
+     * @param mixed    $value
+     *
+     * @return mixed
+     * @throws \JsonException
+     */
+    private static function removeSerialization(WC_Data $object, string $key, $value)
+    {
+        if (is_serialized($value)) {
+            $value = @unserialize(trim($value));
+
+            self::update_meta_data($object, $key, $value);
```

Comment:
Is het hier al ok om die $value te returnen? Misschien niet zo mooi. Anders toch een `else` gebruiken.

Reply:
Nee hij moet nog door dat volgende blok heen, die sluiten elkaar niet uit. Wat ik wel vervelend vind is dat er 2x `update_meta_data` gedaan wordt

Reply:
oke
Je kan ook aan het einde alleen `self::update_meta_data($object, $key, $value); ` doen toch?